Scripts/script-CommonServer.yml

```diff
@@ -98,15 +98,15 @@ script: |-
       }
 
       //in case of headers was not provided (backward compatibility)
-      if (t && (!(headers instanceof Array && headers.length > 0) || !headers)){
+      if ((!headers || !headers instanceof Array) && t && t.length > 0){
```

Comment:
I am not sure this is what happening now. I would explicitly return an empty table no matter what in such a situation, now you continue and something in the rest of the code might not function well because of it